TimePlatformer/Assets/Scripts/PlayerController.cs

```diff
@@ -37,12 +42,26 @@ void Update ()
         if (Input.GetKey(KeyCode.A))
         {
             vel += new Vector2(-1.0f, 0.0f);
+			if(faceRight){
+				faceRight= !faceRight;
+
+				Vector3 revScale = transform.localScale;
```

Comment:
Let's make a private function that handles player flipping as to not have duplicate code.